semcore/scroll-area/src/ScrollArea.jsx

```diff
@@ -71,10 +71,27 @@ class ScrollAreaRoot extends Component {
     if (!this.$container || !this.$wrapper) return size;
     const { scrollWidth, scrollHeight } = this.$container;
     const style = window.getComputedStyle(this.$wrapper);
-    const maxWidth = Number.parseInt(style.getPropertyValue('max-width'));
-    const maxHeight = Number.parseInt(style.getPropertyValue('max-height'));
+    const parent = this.$wrapper.parentElement;
+    const parentStyle = parent ? window.getComputedStyle(parent) : null;
+    const notInAnimation = parentStyle?.getPropertyValue('animation-name') === 'none';
```

Comment:
Maybe you should subscribe to the `animationContext` and recalculate available space on the animation end?